blog-posts/the-modern-code-reviewer-2021.md

```diff
@@ -0,0 +1,219 @@
+
+### **What the reviewee gets**
+
+In-depth discussion of the changes. Many possibilities to receive feedback and iterate changes. As a seasoned developer this is a great way to challenge your ideas and as a less experienced developer you can tackle a complex feature and work on it together with an expert through the PR format.
+
+### **What the reviewee w**on't get
+
+That's the dream package for less experienced and seasoned developers. This is a rare type of review so you won't get them for many changes.
+
+# Which type is the right one for me?
+
+Most people naturally pick the **common review** and don't realize that you can have differently sized reviews. You are easily overwhelmed by the amount of time you have to spend for such a regular code review if the PRs pile up. The common review can take 30 minutes to complete. If you have four reviews you have spend a quarter of your precious working day already.
+
+Other folks exclude themselves from the **common review** because they feel intimidated by their lack of knowledge, the complexity of the code changes or the superior level of the reviewee. The **shallow review** is an excellent instrument to encourage those people in participating in the daily code review routines and start gaining knowledge and experience in that important field. But the shallow review is also a great tool for seasoned developers. You can pick it and quickly confirm a change with the expectation that you did not dug as deep as usually.
+
+Finally the **expert review** is your instrument to ensure an **in-depth review** of content that is pioneering a new approach or introducing security relevant changes.
+
+The moment you understand that you have a choice when you are requested for a code review, you can distribute your time more evenly and steer away from situations that would cause you to be the bottleneck. It's also great to clarify expectations as you can proclaim the way to reviewed. An "I only did a shallow review" makes clear that the PR will deliver a working change without everything being perfect in terms of code quality, architecture and more.
+
+Being team, the knowledge of different code review depths helps you to prevent large backlogs of PRs and frustrated reviewees waiting for some feedback. You have a way to motivate people to take part in the review process who previously lacked the necessary confidence.
+
+# Further improvements
+
+We focused exclusively on the reviewer yet. Being a reviewee, you can enhance the chances that one of the review types will be seen feasible by the reviewer if you make your PR more accessible. Provide a clear description. Repeat parts of the Jira ticket, make the PR small enough, if it's big tell the reviewer why and tell where to focus on. Give clear instructions to run the code and to follow your changes.
```

Comment:
```suggestion
We focused exclusively on the reviewer for now. Being a reviewee, you can increase the chances that one of the review types will be seen feasible by the reviewer if you make your PR more accessible. Provide a clear description. Repeat parts of the Jira ticket, make the PR small enough, if it's big, tell the reviewer why and tell where to focus on. Give clear instructions to run the code and to follow your changes.
```

Reply:
Rephrased and cherry picked your changes.

> We focused solely on the reviewer and how to conduct the actual review. Being the author of a PR (reviewee), you can increase the chances to receive a review if you make your PR more accessible.